source/fundamentals/crud/read-operations/watch.txt

```diff
@@ -146,36 +146,29 @@ For more information on these options, visit the
 Pre- and Post-Images
 ~~~~~~~~~~~~~~~~~~~~
 
-When you perform any CRUD operation on a collection, by default, the corresponding
-change event document contains only the delta of the fields modified
-by the operation. To access the pre-image (the document before a
-change) or the post-image (the document after a change) of a document,
-set the ``FullDocument`` and ``FullDocumentBeforeChange`` fields in the
-options for ``Watch()``.
+When you perform any CRUD operation on a collection, by default, the
+corresponding change event document contains only the delta of the fields modified
+by the operation. You can see the full document before and after a
+change, in addition to the delta, by specifying the ``options``
```

Comment:
S: "Specifying" a parameter sounds a little funny to me. I guess you could say you specified a particular value for the parameter?

```suggestion
change, in addition to the delta, by using the ``options``
```